JM/CF/Scripts/1_Core/CommunityFramework/Logging/CF_Trace.c

```diff
@@ -21,12 +21,19 @@ class CF_Trace
 
 	static string FormatInstance(Class instance)
 	{
-		if (!instance) return "null";
-
-		Object obj;
-		if (Class.CastTo(obj, instance)) return "(" + Object.GetDebugName(obj) + ")";
+		if (!instance)
+		{
+			return "null";
+		}
+		
+		string instanceName;
+		g_Script.CallFunction(instance, "GetDebugNameNative", instanceName, null);
+		if (instanceName == string.Empty)
+		{
+			instanceName = "" + instance;
+		}
 
-		return "" + instance;
+		return instanceName;
```

Comment:
Where do these changes come from now? What is GetDebugNameNative?

Reply:
> GetDebugNameNative

It's a function that is specifically native to the `Object` class. 

> Where do these changes come from now? 

For CF_Trace to work with #72 and #56 I had to move it to `1_Core`. `Object` doesn't exist in `1_Core`.